src/LtiAssignmentsGradesService.php

```diff
@@ -98,6 +98,17 @@ public function createLineitem(LtiLineitem $newLineItem): LtiLineitem
         return new LtiLineitem($createdLineItem['body']);
     }
 
+    public function deleteLineitem(): array
```

Comment:
Do we have any automated tests for this service? If so, we should add a test for this method.

Reply:
It seems we do - I missed it when briefly checking the first time. I was on the fence about adding a test for this since it's so coupled to the implementation that it's not actually testing anything useful. On the flip side, tests are a form of documentation, so as long as we can closely mimic user-level code it's for sure worth having.